app/src/controllers/plugins/uiExtensions/constants.js

```diff
@@ -11,6 +11,7 @@ export const EXTENSION_TYPE_UNIQUE_ERROR_GRID_CELL_COMPONENT =
   'uiExtension:uniqueErrorGridCellComponent';
 export const EXTENSION_TYPE_UNIQUE_ERROR_GRID_HEADER_CELL_COMPONENT =
   'uiExtension:uniqueErrorGridHeaderCellComponent';
+export const EXTENSION_TYPE_LOGIN_BLOCK = 'uiExtension:loginBlock';
 export const EXTENSION_TYPE_LOGIN_PAGE = 'uiExtension:loginPage';
```

Comment:
Can we remove this extension point? Or you plan to use it later?

Reply:
It will be used later